CHANGELOG.md

```diff
@@ -3,6 +3,43 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2024-05-06
+
+### Added
+- Simple RAG Example build a basic Retrieval-Augmented Generation (RAG) system.
```

Comment:
`A simple rag example template showcasing how to build an example from scratch.`
And point to the documentation here.
Add this point to the bottom of list

Reply:
Thanks updated.